src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -311,9 +313,11 @@ async def forward_checksum(self, event: CloudEvent) -> None:
             },
             {event["run_path"]: event.data},
         )
+        # currently clients still need to receive events via ws
```

Comment:
?

Reply:
(what clients?)

Reply:
the forward_model_runner is still intending to use websocket for the communication of checksum events. Then it sounds odd to use the word "currently"?

Reply:
Here, particularly, it has nothing to do with job_dispatch, but it is kept only for the sake of test. As this one only informs the test (described above) that the checksum has been received, since scheduler has not the manifest_queue.